src/overlays/gamestates/ovl_file_choose/z_file_choose.h

```diff
@@ -113,4 +113,13 @@ typedef struct FileChooseContext {
     /* 0x24550 */ s16  unk_24550;
 } FileChooseContext; // size = 0x24558
 
+// TODO: Confirm those values has the same meaning as their OoT counterparts
+typedef enum {
+    /* 0x00 */ PAUSE_0, // PAUSE_ITEM
+    /* 0x01 */ PAUSE_1, // PAUSE_MAP
+    /* 0x02 */ PAUSE_2, // PAUSE_QUEST
+    /* 0x03 */ PAUSE_3, // PAUSE_EQUIP
```

Comment:
This one definitely won't be the same...

Reply:
The respawn enum shouldn't have changed much either, and somehow it changed, so I'm not taking risks here